routes/movies.routes.js

```diff
@@ -0,0 +1,42 @@
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+const router = require("express").Router();
+
+router.get("/movies/create", (req, res, next) => {
+  Celebrity.find()
+    .then((celebritiesFromDB) => {
+      res.render("movies/new-movie", {celebritiesFromDB});
+    })
+    .catch((error) => {
+      console.log("Error getting books from DB", error);
+      next(error);
+    });
+});
+
+router.post("/movies/create", (req, res, next) => {
+    const movieDetails = {
+        title: req.body.title,
+        cast: [req.body.cast],
+        genre: req.body.genre,
+        plot: req.body.plot
+    };
```

Comment:
- Very important -> whatever you send to datbase it has to exactly match the model you have (property names)

In your model you have CELEBRITY, while you are trying to save CAST in the database - which does not exist..

- as mentioned in the morning, you do not need another array, only write this: ` cast: req.body.cast`